test/testUtil.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * testUtil.c
+ *
+ * contains the functions necessary for the testing framework
+ */
+
+#ifndef TEST_UTIL_H_
+#define TEST_UTIL_H_
+
+#include <stdlib.h>
+#include <stdio.h>
+
+#define assertTrue(cond) \
```

Comment:
Please add a comment to indicate why you need this marco. It seems a bit complicated and so we should at least justify its existence.